File: logback-android/src/test/java/android/util/Log.java
package android.util;

public class Log {
  public static int VERBOSE = 2;
  public static int DEBUG = 3;
  public static int INFO = 4;
  public static int WARN = 5;
  public static int ERROR = 6;
  public static int ASSERT = 7;

  public static int d(String tag, String msg) {
    System.out.println("DEBUG: " + tag + ": " + msg);
    return 0;
  }

  public static int i(String tag, String msg) {
    System.out.println("INFO: " + tag + ": " + msg);
    return 0;
  }

  public static int w(String tag, String msg) {
    System.out.println("WARN: " + tag + ": " + msg);
    return 0;
  }

  public static int e(String tag, String msg) {
    System.out.println("ERROR: " + tag + ": " + msg);
    return 0;
  }
}

File: scripts/release/utils/log.ts
import colors from 'picocolors'

export const log = (message = '', ...optionalParams: any[]) =>
  console.log(colors.white(message), ...optionalParams)

export const info = (message = '', ...optionalParams: any[]) =>
  console.info(colors.white(message), ...optionalParams)

export const error = (message = '', ...optionalParams: any[]) =>
  console.error(colors.red(message), ...optionalParams)

export const warn = (message = '', ...optionalParams: any[]) =>
  console.warn(colors.yellow(message), ...optionalParams)

export const verbose = (message = '', ...optionalParams: any[]) =>
  console.debug(colors.dim(message), ...optionalParams)

export const debug = (message = '', ...optionalParams: any[]) =>
  console.debug(colors.dim(message), ...optionalParams)

export const say = (message = '', ...optionalParams: any[]) =>
  console.log(colors.cyan(message), ...optionalParams)

export const success = (message = '', ...optionalParams: any[]) =>
  console.log(colors.green(message), ...optionalParams)
